Wordle.API/Wordle.API/Models/NewScore.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Wordle.Api.Models
+{
+    public record NewScore
```

Comment:
```suggestion
    public record Score
```
If you change your other class to `PlayerDto` then this should be `Score` since you will be posting a new score which will then be used to udpate the corresponding `Player`